test/conformance/chainsaw/validate/clusterpolicy/standard/apicalls/default/pod-bad.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+  namespace: not-default
```

Comment:
this one might be failing because not-default namespace does not exist

Reply:
I will add namespace for this one...